src/lib.rs

```diff
@@ -1353,6 +1356,32 @@ where
     }
 }
 
+impl<K, V, S> Clone for FlurryHashMap<K, V, S>
+where
+    K: Sync + Send + Clone + Hash + Eq,
+    V: Sync + Send + Clone,
+    S: BuildHasher + Clone,
+{
+    fn clone(&self) -> FlurryHashMap<K, V, S> {
+        // TODO: This should be created with the FlurryHashMap::with_capacity_and_hasher once available
+        let cloned_map = Self {
```

Comment:
I had some trouble with the type constrains here. Since `new` and `with_capacity` are implemented for `FlurryHashMap<K, V, RandomState>`. I think (and added a comment) that once the `with_capacity_and_hasher` is implemented it should be use here.

Reply:
Yeah, let's hold off on this until #24 lands, since that'll give us the necessary method.

Reply:
#24 just landed :tada: